content/open-source/sigstore/rekor/how-to-verify-file-signatures-with-rekor-or-curl.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "How to Verify File Signatures with Rekor or curl"
+type: "article"
+description: "Use Rekor or curl to verify non-container software artifacts"
+lead: "Rekor or curl can verify software artifacts beyond containers"
+date: 2022-12-21T15:22:20+01:00
+lastmod: 2022-12-21T15:22:20+01:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "rekor"
+weight: 005
+toc: true
+---
+
+The `rekor-cli` tool or `curl` can be used to verify anything with a signature on a [Rekor transparency log](/open-source/sigstore/rekor/an-introduction-to-rekor/#transparency-log). This tutorial assumes you have the `rekor-cli` tool installed, which you can achieve by following our [How to Install the Rekor CLI](/open-source/sigstore/rekor/how-to-install-rekor/) guide. When verifying a signature using either tool, ensure that you have the `jq` utility installed so that you can parse their output.
+
+We’ll verify a binary artifact, in this case, a copy of [`apko`](/open-source/apko/overview/), which is a command-line tool for building container images using a declarative language based on YAML. The methods in this tutorial apply to any blob file that Cosign has signed with a keyless signature.
```

Comment:
I would make "apko" general here because the link is hard to discern when it's in code format

```suggestion
We’ll verify a binary artifact, in this case, a copy of [apko](/open-source/apko/overview/), which is a tool for building container images using a declarative language based on YAML. The methods in this tutorial apply to any blob file that Cosign has signed with a keyless signature.
```